near-sdk-sim/src/runtime.rs

```diff
@@ -146,17 +146,18 @@ impl Block {
 }
 
 pub struct RuntimeStandalone {
-    genesis: GenesisConfig,
+    pub genesis: GenesisConfig,
     tx_pool: TransactionPool,
     transactions: HashMap<CryptoHash, SignedTransaction>,
     outcomes: HashMap<CryptoHash, ExecutionOutcome>,
     profile: HashMap<CryptoHash, ProfileData>,
-    cur_block: Block,
+    pub cur_block: Block,
     runtime: Runtime,
     tries: ShardTries,
     pending_receipts: Vec<Receipt>,
     epoch_info_provider: Box<dyn EpochInfoProvider>,
     pub last_outcomes: Vec<CryptoHash>,
+    pub runtime_config: Arc<RuntimeConfig>,
```

Comment:
Since this is for tests only, it worth give more control to the user. `cur_block` should be exposed as is and there is no reason to hide it from the user. The main idea is you should have granular control over block time. For example when testing lockup contracts we need to check what would be the balance after a year. For staking pool we want to have access to epoch ID. But if we rely only on produce block advancing this variables, it's hard to write assert statements because you either should replicate math in tests, or just hardcode the output without verifying math. But if you have precise time control, then you can just use round numbers.

Reply:
yeah I was just curious what the interactions would look like to explore if there is an alternative that's easier to maintain/use